src/app/screens/ToDo/components/RemoveSelected/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import {
+  View,
+  TouchableHighlight,
+  Text,
+  StyleSheet,
+} from 'react-native';
+
+function RemoveSelected({ removeSelectedTodos }) {
+  return (
+    <View style={styles.deleteContainer}>
+      <TouchableHighlight onPress={removeSelectedTodos}>
+        <Text style={styles.button}>
+          Remove selected items
+        </Text>
+      </TouchableHighlight>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
```

Comment:
We extract all styles to a separate file call `styles.js` and import them from the components.

Reply:
You can keep the styles in the component for this pr, but keep this in mind for then next card!